src/test/java/Pages/Homework22.java

```diff
@@ -0,0 +1,23 @@
+package Pages;
```

Comment:
Package names are written in all lower case to avoid conflict with the names of classes or interfaces.

For example **pages**

![image](https://user-images.githubusercontent.com/104761482/226459503-6686a595-9722-49df-841a-93aca81b5360.png)